packages/portal/backend/src/controllers/GitHubController.ts

```diff
@@ -327,14 +340,17 @@ export class GitHubController implements IGitHubController {
             }
 
             Log.trace("GitHubController::provisionRepository() - add staff team to repo");
-            const staffTeamNumber = await tc.getTeamNumber(TeamController.STAFF_NAME);
-            Log.trace('GitHubController::provisionRepository(..) - staffTeamNumber: ' + staffTeamNumber);
-            const staffAdd = await this.gha.addTeamToRepo(staffTeamNumber, repoName, 'admin');
+            // const staffTeamNumber = await tc.getTeamNumber(TeamController.STAFF_NAME);
+            // Log.trace('GitHubController::provisionRepository(..) - staffTeamNumber: ' + staffTeamNumber);
+            // const staffAdd = await this.gha.addTeamToRepo(staffTeamNumber, repoName, 'admin');
+            const staffAdd = await this.gha.addTeamToRepo(TeamController.STAFF_NAME, repoName, 'admin');
             Log.trace('GitHubController::provisionRepository(..) - team name: ' + staffAdd.teamName);
 
-            const adminTeamNumber = await tc.getTeamNumber(TeamController.ADMIN_NAME);
-            Log.trace('GitHubController::provisionRepository(..) - adminTeamNumber: ' + adminTeamNumber);
-            const adminAdd = await this.gha.addTeamToRepo(adminTeamNumber, repoName, 'admin');
+            Log.trace("GitHubController::provisionRepository() - add admin team to repo");
+            // const adminTeamNumber = await tc.getTeamNumber(TeamController.ADMIN_NAME);
+            // Log.trace('GitHubController::provisionRepository(..) - adminTeamNumber: ' + adminTeamNumber);
+            // const adminAdd = await this.gha.addTeamToRepo(adminTeamNumber, repoName, 'admin');
+            const adminAdd = await this.gha.addTeamToRepo(TeamController.ADMIN_NAME, repoName, 'admin');
```

Comment:
Whatever is going on here, this is not working :(

Reply:
Not working now, or not working earlier? #[5865fe6](https://github.com/ubccpsc/classy/pull/410/commits/5865fe65c8ae635a57948e2439963e44428df57e) fixed content-type problems that should be resolved now (that could have manifested in this problem because getTeams was not working)

Reply:
Let me try provisioning a team and see what happens

Reply:
OK I tried again and it still doesn't work.

@shizuko-akamoto and I also tried calling this method from a script on our machines

Reply:
I tried putting it back and now it works. The `teamId` in `GitHubActions#addTeamToRepo(teamId: number, repoName: string, permission: string): Promise<GitTeamTuple>;` is actually the team id used by GitHub, not _our_ id. So you do need the query.

Reply:
OH you weren't getting an error because #358 !!! It's responding with 404